.github/workflows/test.yml

```diff
@@ -30,3 +30,76 @@ jobs:
 
       - name: Make Test
         run: make test
+
+  test-kotlin-libraries:
+    name: Test Kotlin Libraries
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - api-level: 28
+            arch-ndk: x86_64
+            arch-cargo: x86_64-linux-android
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Enable KVM
+        run: |
+          echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666", OPTIONS+="static_node=kvm"' | sudo tee /etc/udev/rules.d/99-kvm4all.rules
+          sudo udevadm control --reload-rules
+          sudo udevadm trigger --name-match=kvm
+
+      - name: Cargo toolchain
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: 1.81.0 # Check the version used by Holochain
```

Comment:
I prefer a `rust-toolchain.toml` at the moment. Let the `rustup` provided on the machine install the right toolchain

Reply:
good call https://github.com/holochain/android-service-runtime/pull/57/commits/c66901f050f1815282a913810a2717914e44fe43